### PR TITLE
Ant design setup

### PR DESCRIPTION
**Pull request by m5khan**
antd css and scss plugins added

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,7 +57,10 @@
     "react-redux": "^5.0.7",
     "redux": "^4.0.0",
     "rimraf": "^2.6.2",
+    "sass-loader": "^7.0.1",
+    "scss-loader": "^0.0.1",
```
Comment (michaelneu) [inline package.json]: Why do we need `scss-loader`? The webpack config doesn't use this package, but rather the `sass-loader`.

The yarn lockfile should be affected as well.

Comment (michaelneu) [reply package.json]: I just checked and the package consists of an emtpy `package.json`. We should remove this dependency.

File: src/index.tsx [hunk 2]
```diff
@@ -5,6 +5,9 @@ import { createStore } from "redux";
 import { App } from "./components/app";
 import { rootReducer } from "./reducers";
 
+// css import
```
Comment (tomasruizt) [inline src/index.tsx:8]: no need for this comment

**Review by tomasruizt — APPROVED** [review]
Please make the changes requested. Afterwards, feel free to merge.

**Review by michaelneu — CHANGES_REQUESTED** [review]
Tomas missed requesting changes, so I'll request them to prevent merging this into master.

**Comment by m5khan** [comment]
removed scss-loader

**Review by tomasruizt — APPROVED** [review]

**Review by michaelneu — APPROVED** [review]

**Pull request merged**